graalpy-embedded-pillow/README.md

```diff
@@ -0,0 +1,34 @@
+# Using GraalPy to create an Instance of a Proxied Python Class from a Java Application 
```

Comment:
Let's use a simpler name for a demo:
```markdown
# Image Manipulation Java Application with Python Pillow Library Embedded

This application demonstrates how to use GraalPy to create an instance of a proxied Python class from a Java application, built with Maven.
```

Reply:
The other variant for a title:
```
# Image Manipulation Demo Using Java and GraalPy
```

Reply:
How about "Image Manipulation using a Java Application with an Embedded Python Library"

Reply:
using > Using